streams/src/test/java/org/apache/kafka/streams/integration/AdjustStreamThreadCountTest.java

```diff
@@ -180,6 +182,19 @@ public void shouldRemoveStreamThread() throws Exception {
         }
```

Comment:
One last thing, can you add a version of the `shouldRemoveStreamThread()` test that uses static membership?

Reply:
yes